include/NovelRT/Input/InteractionService.h

```diff
@@ -13,22 +13,23 @@ namespace NovelRT::Input {
     friend class Windowing::WindowingService; //I get this looks weird but its because GLFW treats the window as this system as well as the window.
 
   private:
-    NovelRunner* const _runner;
+    static inline const uint32_t INPUT_BUFFER_COUNT = 2;
```

Comment:
Why hardcode to 2?

Reply:
I just picked two as a standard amount. No real reason.

Reply:
Is this meant to be the same as the render buffer? Should it be configurable?

Reply:
Maybe? However if we want an API for configuring it that's out of scope for this ticket and we'll need to open another issue, this bug has been hanging around for too long as it is.

Reply:
Is it out of scope? It's just taking the count in the constructor, rather than taking a constant here.

Reply:
It's an API change, which is my point. I dislike mixing API changes within those if possible and this branch purely exists to fix the bug, but I can introduce it if you want. It's not a big deal really I suppose.

Reply:
It's not a big deal either way. I was just commenting that this is introducing a new hardcoded concept and there isn't a corresponding issue tracking that.